### PR TITLE
Feature/setShortcut

### PR DESCRIPTION
**Pull request by andreGarvin**
@rdepena @datamadic @whyn07m3

### CODE REVIEW

File: src/api/application/application.ts [hunk 1]
```diff
@@ -212,6 +218,16 @@ export class Application extends Base {
     }
 
     /**
+     * Sets new application's shortcut configuration.
+     * @param { Object } config New application's shortcut configuration.
+     * @return {Promise.<void>}
+     * @tutorial Application.setShortcuts
+     */
+    public setShortcuts(config: ConfigInterface): Promise<void> {
+        return this.wire.sendAction('set-shortcuts', Object.assign({}, this.identity, config)).then(() => undefined);
```
Comment (whyn07m3) [inline src/api/application/application.ts]: • Best to put `Object.assign({}, this.identity, config)` on a separate line and assign it to a variable for better readability
• Is `.then(() => undefined);` needed here?

File: tutorials/Application.setShortcuts.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Sets new application's shortcut configuration
+
+# Example
+```js
+async function createWin(config) {
+		const app = await fin.Application.create({
+			uuid: 'app-1',
+			name: 'myApp',
```
Comment (whyn07m3) [inline tutorials/Application.setShortcuts.md]: Shouldn't `uuid` & `name` be the same?

File: tutorials/Application.setShortcuts.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Sets new application's shortcut configuration
+
+# Example
+```js
+async function createWin(config) {
```
Comment (whyn07m3) [inline tutorials/Application.setShortcuts.md]: Misleading function name. This function creates an application, but returns its main window.

File: tutorials/Application.setShortcuts.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+		const app = await fin.Application.create({
```
Comment (whyn07m3) [inline tutorials/Application.setShortcuts.md]: I think spacings are off here & in other places

File: tutorials/Application.setShortcuts.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    const app = await createWin();
+    return await app.setShortcuts(config);
+}
+
+setShortcuts({
+		desktop: true,
+		startMenu: false,
+		systemStartup: true
+}).then(() => console.log('Shortcuts are set.')).catch(err => console.log(err));
+```
```
Comment (whyn07m3) [inline tutorials/Application.setShortcuts.md]: I'd rewrite the whole thing to:

```
const app = await fin.Application.create({
   uuid: 'app',
   name: 'app',
   url: 'https://openfin.co',
   autoShow: true
});

await app.run();

app.setShortcuts({
   desktop: true,
   startMenu: false,
   systemStartup: true
})
   .then(() => console.log('Application shortcuts has been set'))
   .catch(console.error);
```

File: tutorials/Application.setShortcuts.md (hunk 2 above)
Comment (andreGarvin) [reply tutorials/Application.setShortcuts.md]: I would not matter because it does not really because the application `uuid` is given

**Pull request merged**